libnmstate/nm/team.py

```diff
@@ -102,7 +102,9 @@ def get_info(device):
 
 
 def _convert_teamd_config_to_nmstate_config(team_config):
-    team_config.pop("device")
+    if team_config.get(TEAMD_JSON_DEVICE):
+        team_config.pop(TEAMD_JSON_DEVICE)
```

Comment:
Please use pop(key, None) to ignore missing key

Reply:
Done, thanks! :-)